C#/playbook.yaml

```diff
@@ -0,0 +1,51 @@
+---
+- hosts: 127.0.0.1
+  become: true
+  tasks:
+    - apt: update_cache=yes
+    
+    - name: Add the Microsoft package signing key to your list of trusted keys and add the package repository
+      get_url: 
+        url="https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb"
+        dest="/home/codio/packages-microsoft-prod.deb"
+
+    - name: Install packages-microsoft-prod
+      apt: deb="/home/codio/packages-microsoft-prod.deb"
+      sudo: true
+    
+    - name: Run the equivalent of "apt-get update" as a separate step
+      apt:
+        update_cache: yes
```

Comment:
use apt from url like:
```
    - name: Install OmniDB
      apt:
        deb: https://github.com/MaximKraev/OmniDB/releases/download/2.17.0-1/omnidb-server_2.17.0-debian-amd64.deb
```

also you do not need update_cache separately can be added to the next command